en/intro/conventions.rst

```diff
@@ -142,6 +142,18 @@ Now that you've been introduced to CakePHP's fundamentals, you might try a run
 through the :doc:`/tutorials-and-examples/bookmarks/intro` to see how things fit
 together.
 
+Core Naming Conventions
```

Comment:
Should I put it in the ORM initial chapers as ORM conventions?
Only maybe FormHelper may also juggle field, column and property, if at all..?!